newIDE/app/src/InstancesEditor/index.js

```diff
@@ -134,7 +139,6 @@ export default class InstancesEditorContainer extends Component {
     this._mountEditorComponents(this.props);
     this._renderScene();
   }
-
```

Comment:
sorry, got a bit carried away 😄

Reply:
I know that I'm nitpicking but there are two extra spaces that are useless ;) Don't do anything I will remove these.
In fact instead of doing this by hand, you should really install Prettier extension for VSCode, then run "format document":

![image](https://user-images.githubusercontent.com/1280130/39308815-546f52b2-495e-11e8-84c2-b66b7a4d4f85.png)

It reformat all the code of the file, turning mess like this:
![image](https://user-images.githubusercontent.com/1280130/39308860-71d65152-495e-11e8-909e-68e838bdb8e1.png)

in code which is follow strict formatting rules and ensure that the whole codebase is consistent: 
![image](https://user-images.githubusercontent.com/1280130/39308893-8d15ebda-495e-11e8-9944-87259bd11d5e.png)